client/src/components/TextInputComponent/TextInput.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./TextInput.css";
+
+class TextInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: "" };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
```

Comment:
Close! Sorry this wasn't clear: the `handleChange` function and `value` should be passed in as a prop from the Storybook file. This component should probably be just a functional component.